packaging/docker_sys_config.sh

```diff
@@ -11,18 +11,6 @@ sed -i \
     -e 's/^#RateLimitBurst=.*/RateLimitBurst=0/' \
     /etc/systemd/journald.conf
 
-cat > /etc/NetworkManager/conf.d/97-no-managed.conf <<EOF
-[device]
-match-device=*
-managed=0
-EOF
```

Comment:
The new config-server is not including this option.

I think you should mention that we drop this one explicitly in the commit message. It will be even better to split the drop in a commit before this one so we could revert it back if we find again that the tests are not stable.